docs/contributing.md

```diff
@@ -25,15 +25,23 @@ Please submit bug reports, feature requests and feedback as issues in GitHub: [h
 
 Please see the information provided in [SECURITY.md](SECURITY.md)
 
+## Creating Your Own Fork  of `scores` for the First Time
+
+Unless you are an advanced Git user, we would recommend you follow this process:
+
+1. First (i.e. **before** cloning the `scores` repository) create your own fork using the GitHub web user interface.
+2. Clone **your fork**. (Do not clone github.com/nci/scores).
+3. Immediately create a new local branch, with a command such as `git checkout -b branch_name`.
+
```

Comment:
Consider a line to periodically sync your fork

Reply:
It could possibly fit into the section just below

Reply:
I have added a commit to the PR to add some information on this to the PR workflow. Let me know if that seems sufficient.

Reply:
Looks good - In addition it may be good to encourage people to sync their fork *before* they create a new feature branch to minimise the amount of rebasing etc.

Reply:
While that's good advice, I feel like it may be stepping over into a more detailed guide that we don't really have the ability to provide. Ideally, it would be nice to link to some general resource that hopefully already exist out there. Feel free to do some looking around and raise a new PR if you find some good ones.